ch07.asciidoc

```diff
@@ -167,7 +167,7 @@ which, as you can see, is much shorter. Instead of "pay to this 5-key multisigna
 <Sig1> <Sig2> <2 PK1 PK2 PK3 PK4 PK5 5 CHECKMULTISIG>
 ----
 
-The two scripts are combined in two stages. First, the redeem script is checked against the locking script to make sure the hash matches:
+The two scripts are combined in two stages. First, the redeem script is checked against the unlocking script to make sure the hash matches:
```

Comment:
I believe this is correct as written. The definition of "locking script" comes from Chapter 6:

> A cryptographic puzzle that determines the conditions required to spend the
output [...] The cryptographic puzzle is also known as a *locking script*, a witness script, or a `scriptPubKey`.

And in Chapter 7:

> the complex script that details the conditions for spending the output (redeem script) is not presented in the locking script. Instead, only a hash of it is in the locking script and *the redeem script itself is presented later, as part of the unlocking script* when the output is spent.

So here we do indeed intend to check the redeem script (part of the unlocking script) against the original *locking script*. Checking part of the unlocking script against the *unlocking script* itself wouldn't make sense.